cert/cert_v2.asn1

```diff
@@ -0,0 +1,38 @@
+Nebula DEFINITIONS AUTOMATIC TAGS ::= BEGIN
+
+Name ::= UTF8String (SIZE (1..253))
+Time ::= INTEGER (0..18446744073709551615) -- uint64 maximum
+Network ::= OCTET STRING (SIZE (5,17)) -- IP addresses are 4 or 16 bytes + 1 byte for the prefix length
+Curve ::= ENUMERATED {
+    curve25519 (0),
+    p256 (1)
+}
+
+-- The maximum size of a certificate must not exceed 65536 bytes
+Certificate ::= SEQUENCE {
+    details OCTET STRING,
+    curve Curve DEFAULT curve25519,
+    publicKey OCTET STRING,
+    -- signature(details + curve + publicKey) using the appropriate method for curve
+    signature OCTET STRING
+}
+
+Details ::= SEQUENCE {
+    name Name,
+
+    -- At least 1 ipv4 or ipv6 address must be present if isCA is false
+    networks SEQUENCE OF Network,
+    unsafeNetworks SEQUENCE OF Network OPTIONAL,
```

Comment:
Very happy to see this getting renamed from `-ips` and `-subnets` which were a bit confusing (many people believed subnets was essentially the CIDR part of `-ips`.) That being said, calling them both "networks" might also be confusing. I think I'd lean towards simply calling this `unsafeRoutes` to match the config option.

Reply:
I'm not opposed to calling these `unsafeRoutes`, it doesn't quite feel right though since this field is describing a network that can be used to configure unsafe routes on another host.